.gitignore

```diff
@@ -42,6 +42,12 @@ project.yaml
 
 acceptance/hosts.yaml
 
+modules/aggregate
```

Comment:
TODO: Check that this needs to be done

Reply:
`aggregate`, `canary`, `puppetdb_fact`, `secure_env_vars` shouldn't be ignored because those are included in the Bolt repo.

Reply:
Removed and will address in a separate work item